tutorials/simulators/CustomImage.md

```diff
@@ -0,0 +1,378 @@
+- **Simulation time (`N 41`)**: 25.0
+- **Paraview results rate (`P 30`)**: 0.01
+
+
+### Step 2: Running the Simulation
+
+#### a. Configure and Start Machine
+
+1. **Pick your machine**:
+
+    Our `CustomImage` simulator will work with any machine type. So, we decided
+    to use the same machine as in the previous tutorial, the `c2d-highcpu-112`
+    with the same configuration.
+
+	```python
+	import inductiva
+	machine_group = inductiva.resources.MachineGroup(
+						machine_type="c2d-highcpu-112",
+						spot=True,
+						data_disk_gb=20,
+						auto_resize_disk_max_gb=250)
+	```
+	**Note**: `spot` machines are a lot cheaper but can be terminated by the
+	provider if needed.
+
+2. **Start your machine**
+	```python
+	machine_group.start()
+	```
+
+#### b. Define Simulation Inputs
+
+1. **Specify Simulation Directory**:
+	Let's start by defining a variable that points to the `10_2_3D_Dam_Break_with_Obstacle`
+	folder where all your simulation files are located.
+
+	```python
+	input_dir = "./10_2_3D_Dam_Break_with_Obstacle"
+	```
+
+2. **Choose Your Simulator**:
+
+This step is straightforward. We will be using the `CustomImage` simulator to
+run our custom Docker image. The `container_image` parameter points to the
+Docker image we want to use.
+
+```python
+customImage = inductiva.simulators.CustomImage(
+    container_image="docker://inductiva/kutu:reef3d_v24.02")
+```
+
+3. **Define Commands**:
+    Our `CustomImage` simulator accepts a list of commands to run inside the
+    container. We will be running two commands: `DiveMESH` and `REEF3D`, present
+    in the Reef3D image.
+
+    ```python
+    # Define the MPI configuration
+    mpi_config = inductiva.commands.MPIConfig("4.1.6", np=56, use_hwthread_cpus=False)
+
+    # Define the commands to run with the MPI configuration
+    command = inductiva.commands.Command("/REEF3D/bin/REEF3D", mpi_config=mpi_config)
+
+    # List of commands to run our simulation
+    commands = [
+        "/DIVEMesh/bin/DiveMESH",
+        command
+    ]
+    ```
+
+    Lets talk about the `MPIConfig` for a command. The `MPIConfig` class allows you
+    to run your command with a specific MPI version. In this case, we are using
+    OpenMPI version 4.1.6 with 56 processes. Creating your command with an `MPIConfig`
+    will parallelize said command across the specified number of processes.
+
+    **Note:** Not all commands can be parallelized. Make sure the command you are
+    running supports parallelization.
+
+#### c. Run Your Simulation
+
+1. **Run the simulation**:
+	We now have everything we need to run our simulation.
+	```python
+	task = customImage.run(
+        input_dir=input_dir,
+        commands=commands,
+        on=machine_group,
+        n_vcpus=56,
+        use_hwthread=False,
+        storage_dir="3D_dam_break_with_obstacle")
+	```
+
+	As you can see, this is very similar to the Reef3D example. The only difference
+    is that here we need to specify the commands to run the simulation.
```

Comment:
The only difference here is that we need to specify the specific commands required to run the simulation. @Pbarbosa92